generator/code_generator.go

```diff
@@ -2454,6 +2461,17 @@ func (d *VariableDefinition) RequestType() string {
 	return ""
 }
 
+func (d *VariableDefinition) LogValueRequestType() string {
+	expr := d.VariableDefinition.Expr
+	switch {
+	case expr.Call != nil:
+		return fullMessageName(expr.Call.Request.Type)
+	case expr.Message != nil:
```

Comment:
Since it will always be called under `$def.IsCall` in template, is this condition actually not evaluated ?

Reply:
Yes, this condition is not actually evaluated. I did the same as RequestType(), but we can remove it if not needed.

Reply:
Sure, Let's remove it then !